skimage/color/colorconv.py

```diff
@@ -1661,16 +1661,18 @@ def combine_stains(stains, conv_matrix, *, channel_axis=-1):
 
 @channel_as_last_axis()
 def lab2lch(lab, *, channel_axis=-1):
-    """CIE-LAB to CIE-LCH color space conversion.
+    """Convert image in CIE-LAB to CIE-LCh color space.
 
-    LCH is the cylindrical representation of the LAB (Cartesian) colorspace
+    CIE-LCh is the cylindrical representation of CIE-LAB (Cartesian).
```

Comment:
```suggestion
    CIE-LCh is the cylindrical representation of the CIE-LAB (Cartesian) color space.
```
Wikipedia tells me that the "color space" part is not included in the abbreviation?

Also just curious, why do you use CIE-LCh and CIE-LAB over CIELCh and CIELAB as used by Wikipedia ([example](https://en.wikipedia.org/wiki/HCL_color_space))? Can't be http://www.easyrgb.com/en/math.php because that using different names too.

Reply:
It's somewhat arbitrary but, after pondering different options, I decided to go for this compromise:

* There were several instances of 'CIE-LAB' in `skimage/color/colorconv.py` (before #6688) so I've taken this legacy into account;
* Reference http://www.easyrgb.com/en/math.php hyphenates these colorspace acronyms, so even if we don't follow *exactly* their convention, it gives weight to the possible choice of hyphenating;
* I'd be happy to go the Wikipedia way, but since 'CIE' acts like a prefix, we might as well use a hyphen for the sake of clarity and readability. Likewise, you may write either 'postmodernism' or 'post-modernism' -- both are correct, but 'post-modernism' is more easily parseable at a glance. I try to make it as comfortable as possible for the reader/user and for the future maintainer :slightly_smiling_face:

Reply:
Oh! Ok, it makes the sentence absolutely clear, so I've committed your suggestion. Even if "CIE-LAB" doesn't literally stand for "CIE-LAB color space," it does 'semantically' and my wording was just being elliptical (shorthand style). But, again, let's be clear and unambiguous, so thanks for your suggestion!